client/src/components/navbar-component/navbar.module.css

```diff
@@ -0,0 +1,61 @@
+/* Navbar Styling */
+
+.navbar {
+    /*
+    - Change background color to orange
+    - Turn the navbar into a flex container w/ space between items
+    - Align the items vertically across the center
+    - Add a bottom margin of 35px
+    - Add a padding of 25px to thicken navbar
+    */
+    background-color: #FFFDF2;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 25px;
+  }
+  
+.title a {
+    /*
+    - Change font color to white
+    - Set font size to 35px
+    - Add a left margin of 25px
+    - Get rid of the link underline with text-decoration
+    - Bold the text
+    */
+    color: #14452F;
+    font-size: 35px;
+    text-decoration: none;
+    font-weight: bold;
+  }
+  
+.nav_buttons a {
+    /*
+    - Change font color to white
+    - Set font size to 25px
+    - Add a right margin of 16px
+    - Get rid of the link underline with text-decoration
+    */
+    color: #14452F;
+    font-size: 25px;
+    margin-right: 16px;
+    text-decoration: none;
+  }
+
+.calendar {
+  filter: invert(19%) sepia(19%) saturate(1428%) hue-rotate(101deg) brightness(100%) contrast(92%);
+  width: 65px;
+  height: 65px;
+  border-radius: 0%;
+  overflow: hidden;
+
+  margin-left: 1100px;
```

Comment:
I think you should find another way to position the calendar since we want the design to be responsive and pixels usually make it so it looks different on other devices. I suggest using either % or using flexbox or grid for the spacing!